nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/SameDiff.java

```diff
@@ -1427,6 +1429,52 @@ public long numElements() {
         return new ArrayList<>(variableMap().values());
     }
 
+    /**
+     * Get the names of variables (if any) that have been marked as loss variables to be minimized.<br>
+     * Variables can be marked as loss variables in a few different ways:<br>
+     * (a) Losses are automatically added when creating loss functions via {@link #sd()}<br>
+     * (b) Via {@link #setLossVariables(String...)}, @link #addLossVariable(String)} or {@link SDVariable#markAsLoss()}<br>
+     * (c) Via {@link TrainingConfig#setLossVariables(List)}<br>
+     */
+    public List<String> getLossVariables(){
+        return this.lossVariables;
```

Comment:
do we want to return a mutable version of the list here?

Reply:
Probably not.